### DISCUSSION TITLE
landing page prebuilt installer

### DISCUSSION THREAD

**Discussion by Bgregg93** [nodejs/nodejs.org#7284] [Ideas]
I noticed that it is generally favoured to use the package manager for installing node, should we possibly redirect the landing page download button to the package manager installation steps instead?

**Comment by AugustinMauroy** [comment]
Hmmm -1 for that because the download button on the landing page is aimed at new node users who want to discover and experience more iron on the downlaod page.

**Reply by Bgregg93** [reply]
This makes sense I was having a conversation with another user over the website and we questioned if that was overlooked or not I will close this discussion.
